packages/example/pages/index.tsx

```diff
@@ -243,9 +243,17 @@ const Example = () => {
                 <div>Transaction: {JSON.stringify(transactionData)}</div>
               )}
               {transactionError && <div>Error sending transaction</div>}
-              {signingData && <div>Data Signature: {signingData}</div>}
+              {signingData && (
+                <div style={{ wordBreak: 'break-all' }}>
```

Comment:
I added this so that the layout doesn't break on mobile when signing a message.

Reply:
Thanks this was sloppy by me.